src/test/java/org/opensearch/ad/AbstractADTest.java

```diff
@@ -212,7 +216,7 @@ protected static void setUpThreadPool(String name) {
                 AnomalyDetectorPlugin.AD_THREAD_POOL_NAME,
                 1,
                 1000,
-                "opendistro.ad." + AnomalyDetectorPlugin.AD_THREAD_POOL_NAME
+                "opensearch.ad." + AnomalyDetectorPlugin.AD_THREAD_POOL_NAME
```

Comment:
Nice find. Can you grep in your final codebase that has all of the changes, that there are no more references of `opendistro`?

Reply:
There are quite a few of opendistro like

src/main/java/org/opensearch/ad/constant/CommonName.java
34:    public static final String CHECKPOINT_INDEX_NAME = ".opendistro-anomaly-checkpoints";
36:    public static final String DETECTION_STATE_INDEX = ".opendistro-anomaly-detection-state";
40:    public static final String ANOMALY_RESULT_INDEX_ALIAS = ".opendistro-anomaly-results";

According to infra team, our setting will have backward compatibility to opendistro, while our indices will use keep using opendistro.

Reply:
Understood, agreed the BWC vars and the indices will remain `opendistro`, was more referring to other places that can be changed like the thread pool name above.

Reply:
right, fixed a few in our docs and comments.  Please check the new commit.

Reply:
Not sure what commit you are referring to?

Reply:
this commit: https://github.com/opensearch-project/anomaly-detection/pull/77/commits/4cf9c7a24337dd27b29bbfc77596091c6281778d